rules/aip0192/has_comments_test.go

```diff
@@ -20,17 +20,52 @@ import (
 	"github.com/googleapis/api-linter/rules/internal/testutils"
 )
 
-func TestHasComments(t *testing.T) {
-	file := testutils.ParseProto3String(t, `
+func TestFieldHasComments(t *testing.T) {
```

Comment:
Go's practice is to name the test based on the function being tested. Test{Function}, which in this case was TestHasComments. The test cases within a test function should describe the specific scenario being tested, where a test case is an item in the "table driven test". I split it out to Field and Message because they tested that function in specific contexts (rather than make one big test that did too much). I'd prefer not to rename these test functions, unless it was just moving it around e.g. TestHasCommentsField TestHasCommentsMessage or something of the like.